chaosazure/vmss/actions.py

```diff
@@ -7,6 +8,7 @@
 from chaosazure import init_client
 from chaosazure.rgraph.resource_graph import fetch_resources
 from chaosazure.vmss.constants import RES_TYPE_VMSS
+from chaosazure.machine.constants import OS_LINUX, OS_WINDOWS
```

Comment:
This should be ordered alphabetically. We probably could add [isort](https://pypi.org/project/isort/) to the package one day.